monero-outreach-docs/translations/fr/archive/triptych_fr.md

```diff
@@ -0,0 +1,43 @@
+
+La manière dont les transaction-leurres TXO sont sélectionnées a partir de la chaine de blocs s’est améliorée au fil du temps. Au tout début, l'intégration de transaction-leurres - bien que faisant partie du système - restait facultative, et les leurres étaient sélectionnées uniformément sur l'historique entière des transactions. Avec le temps, le nombre de transaction-leurres grandit et devint obligatoire, pour atteindre aujourd'hui exactement 10 transaction-leurres par ordre d’exécution. De même, le procédé statistique de sélection fut affiné au fil du temps pour lui donner plus de complexité grâce au "tailored random process" (un algorithme basé sur le procédé de sélection pour l’exécution de l’ordre d’après une théorie statistique de distribution gamma particulière, que nous utilisons encore a ce jour).
+
+Les algorithmes du "core ring signature" ont également été améliorés: la publication de proposition originelle ("whitepaper") de CryptoNote présentait un processus de signature de cercle différent de celui que nous employons actuellement, qui est bâti sur l’utilisation des signatures "Multilayered Linkable Spontaneous Anonymous Group" (MLSAG). Une amélioration substantielle, nommée signatures "Compact Linkable Spontaneous Anonymous Group" (CLSAG), est attendue prochainement, et promet de réduire d’environ un quart le poids (en termes de données) des transactions et d’un dixième leur temps de confirmation (pour une transaction de deux entrées et deux sorties).
+
+Toutes ces améliorations ont donné à Monero de puissantes capacités d’anonymisation. A chaque mouvement de Monero, tout TXO sortant prend la forme d’une clé publique TXO de couverture, sans lien cryptographique avec l’adresse émettrice, mêlée a 10 autres clé publiques-leurres, sélectionnées pour leur vraisemblance. Cela rend toute surveillance extrêmement difficile, tout en assurant une rapidité et une fluidité d’exécution du logiciel de Monero, et offrant une chaine de blocs de taille et de vitesse acceptable.
+
+### _Les Défis_
+
+Néanmoins, la technologie obfusquant l’émetteur n’est pas optimale. Dix transactions-leurres ne sont pas toujours suffisantes parce que certains TXO aléatoires pourraient être identifiés comme étant des leurres. Ils auraient déjà pu servir à des transactions identifiées précédemment. L’existence de listes noires pourrait poser problème et offrir à des adversaires motivés une arme potentielle en prenant l’initiative de les ajouter à la liste de leurs portefeuilles. Il y a également d’autres astuces pour identifier les leurres, en liant les TXO aux échanges ou aux pools de mineurs de Monero, en estimant leur usage dans d’autres transferts et en évaluant des probabilités statistiques temporelles.
```

Comment:
Ils auraient déjà pu servir à des transactions identifiées précédemment. >>> Ils ont peut-être déjà servi pour des transactions déjà identifiées et révélé par le passé

Reason: Choice of translation. Accuracy.

Reply:
Integrated